### PR TITLE
Fix "Reindent" option for sqlformat.

### PR DESCRIPTION
**Pull request by pengjunjie1207**
### What does this implement/fix? Explain your changes.

For sqlparse-0.2.3 and sqlparse-0.2.4, beautifying sql will throw "sqlformat: error: argument -r/--reindent: ignored explicit argument". I remove the param asignment in the argument and change if condition.

### Does this close any currently open issues?

No.

### Any other comments?

The code works with user's config about "Reindent" in settings->atom-beautify->sql and I hava tested it in both Ubuntu and Windows.

### Checklist

Check all those that are applicable and complete.

It is the first time I use Travis CI. It passed linux and failed osx, saying "Couldn't set selectedTextBackgroundColor".

- [x] Merged with latest `master` branch
- [ ] Regenerate documentation with `npm run docs`
- [x] Add change details to `CHANGELOG.md` under "Next" section
- [ ] Added examples for testing to [examples/ directory](examples/)
- [ ] Travis CI passes (Mac support)
- [x] AppVeyor passes (Windows support)

### CODE REVIEW

**Comment by Glavin001** [comment]
Related to https://github.com/Glavin001/atom-beautify/pull/1926

File: src/beautifiers/sqlformat.coffee [hunk 1]
```diff
@@ -17,7 +17,7 @@ module.exports = class Sqlformat extends Beautifier
   beautify: (text, language, options) ->
     @run("sqlformat", [
       @tempFile("input", text)
-      "--reindent=#{options.reindent}" if options.reindent?
+      "--reindent" if (options.reindent? && options.reindent == true)
```
Comment (Glavin001) [inline src/beautifiers/sqlformat.coffee]: I think this should be:

```coffeescript
"--reindent" if options.reindent is true
```

Which is the following in JavaScript:

```javascript
if (options.reindent === true) {
  "--reindent";
}
```

Comment (pengjunjie1207) [reply src/beautifiers/sqlformat.coffee]: What about the situation that options.reindent is null or undefined? Or it is impossible to meet that situation.

Comment (Glavin001) [reply src/beautifiers/sqlformat.coffee]: It could happen and then it would not add `--reindent` option at all. Is this the desired behaviour?

**Comment by Glavin001** [comment]
@szeck87 Do you recall why the SQL test case was disabled?
See https://github.com/Glavin001/atom-beautify/tree/master/examples/nested-jsbeautifyrc/sql/original
We have `sqlparse` installed for Travis CI: https://github.com/Glavin001/atom-beautify/blob/master/requirements.txt#L3

**Comment by Glavin001** [comment]
Published to v0.32.2

**Pull request merged**